docs/user-guide/examples.md

```diff
@@ -296,3 +296,76 @@ defaultEntryPoints = ["http"]
 IdleTimeout = "360s"
 ProvidersThrottleDuration = "5s"
 ```
+
+## Securing Ping Health Check
+The `/ping` health-check URL is enabled together with the web admin panel, enabled with the command-line `--web` or config file option `[web]`. Thus, if you have a regular path for `/foo` and an entrypoint on `:80`, you would access them as follows:
```

Comment:
Could add an empty line between the before this line?
Could you put one sentence by line?

Reply:
Sure, updating now. Out of curiosity, why? I have never seen markdown done with a CRLF between each sentence.

Reply:
In Markdown, `\n` between two sentences is replaced by a simple ` ` (Expect in Github 😉 )

https://daringfireball.net/projects/markdown/syntax#p